src/apprt/gtk/Surface.zig

```diff
@@ -1533,15 +1551,17 @@ fn gtkMouseScroll(
     const scaled = self.scaledCoordinates(x, y);
 
     // GTK doesn't support any of the scroll mods.
-    const scroll_mods: input.ScrollMods = .{};
+    const scroll_mods: input.ScrollMods = .{ .precision = self.precision_scroll };
+    // Multiply precision scrolls by 10 to get a better response from touchpad scrolling
+    const multiplier: f64 = if (self.precision_scroll) 10 else 1;
```

Comment:
I used 10, but it does feel a little high to me. I'm not sure what other people think for scroll behavior here.